test/views/classofbeneficiary/remove/WhenRemovedViewSpec.scala

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.classofbeneficiary.remove
+
+import java.time.LocalDate
+
+import forms.DateRemovedFromTrustFormProvider
+import models.Name
+import play.api.data.Form
+import play.twirl.api.HtmlFormat
+import views.behaviours.QuestionViewBehaviours
+import views.html.classofbeneficiary.remove.WhenRemovedView
+
+class WhenRemovedViewSpec extends QuestionViewBehaviours[LocalDate] {
+
+  val messageKeyPrefix = "classOfBeneficiary.whenRemoved"
```

Comment:
This didn't catch the trustee typo in the text.

Reply:
See above, it's not used.